.github/workflows/build-artifact.yaml

```
@@ -0,0 +1,66 @@
+      #         Clean up files older than 7 days
+      - name: Checkout Artifacts Branch
+        run: |
+          git fetch origin artifacts
+          git checkout -B artifacts origin/artifacts || git checkout --orphan artifacts
+          find dist -mtime +7 -exec rm {} \;
+
+      # Step 5: Add the tarball to the artifacts branch
+      - name: Add and Commit Tarball
+        run: |
+          cd dist
+          git add flightctl-${{ github.ref }}-vendor.tar.bz2 flightctl-${{ github.ref }}.tar.gz     
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)